guide/tests/travel_agency_service_parallel_test.bal

```diff
@@ -21,17 +21,6 @@ import ballerina/test;
 function beforeFunc () {
     // Start the 'travelAgencyService' before running the test
     _ = test:startServices("travel_agency");
-
```

Comment:
Shall we remove this tests folder altogether? (remove it and create a git issue to add them back linking the removing commit) (need to remove this because right now build command fails with top-level integration tests

Reply:
done